system-parachains/asset-hubs/asset-hub-kusama/src/staking/mod.rs

```diff
@@ -416,9 +519,206 @@ where
 	RuntimeCall: From<LocalCall>,
 {
 	fn create_bare(call: RuntimeCall) -> UncheckedExtrinsic {
-		// TODO: double check. the only way I found to access the `new_bare` method.
-		// `UncheckedExtrinsic` is not generic::UncheckedExtrinsic, its wrapped by pallet revive's
-		// type `UncheckedExtrinsic`.
 		<UncheckedExtrinsic as TypeInfo>::Identity::new_bare(call).into()
 	}
 }
+
+pub struct InitiateStakingAsync<const EXPECTED_SPEC: u32>;
+impl<const EXPECTED_SPEC: u32> frame_support::traits::OnRuntimeUpgrade
+	for InitiateStakingAsync<EXPECTED_SPEC>
+{
+	fn on_runtime_upgrade() -> Weight {
+		if crate::VERSION.spec_version == EXPECTED_SPEC {
+			use pallet_election_provider_multi_block::verifier::Verifier;
+			// set parity staking miner as the invulnerable submitter in `multi-block`.
+			// https://kusama.subscan.io/account/GtGGqmjQeRt7Q5ggrjmSHsEEfeXUMvPuF8mLun2ApaiotVr
+			let acc = hex_literal::hex!(
+				"bea06e6ad606b2a80822a72aaae84a9a80bec27f1beef1880ad4970b72227601"
+			);
+			if let Ok(bounded) = BoundedVec::<AccountId, _>::try_from(vec![acc.into()]) {
+				multi_block::signed::Invulnerables::<Runtime>::put(bounded);
+			}
+
+			// set the minimum score for the election, as per the kusama RC state.
+			//
+			// This value is set from block [29,940,247](https://dev.papi.how/explorer/0xf8e2599cd04321369810cd6b4c520f4bc3a8f08f76089d0e467d4a0967179a94#networkId=kusama&endpoint=wss%3A%2F%2Frpc.ibp.network%2Fkusama) of Kusama RC.
+			// Recent election scores in Kusama can be found on:
+			// https://kusama.subscan.io/event?page=1&time_dimension=date&module=electionprovidermultiphase&event_id=electionfinalized
+			//
+			// The last example, at block [29939392](https://kusama.subscan.io/event/29939392-0) being:
+			//
+			// * minimal_stake: 6543_701_618_936_726 (2.12x the minimum -- 6.5k KSM)
+			// * sum_stake: 8_062_560_594_210_938_663 (2.3x the minimum -- 8M KSM)
+			// * sum_stake_squared: 67_504_538_161_651_736_253_970_267_717_229_279 (0.8 the minimum,
+			//   the lower the better)
+			let minimum_score = sp_npos_elections::ElectionScore {
+				minimal_stake: 2957640724907066,
+				sum_stake: 3471819933857856584,
+				sum_stake_squared: 78133097080615021100202963085417458,
```

Comment:
So this value should be larger than recent election `sum_stake_squared`? Recent election event here https://kusama.subscan.io/event/29967817-0 looks like about 2^115.7, whereas the value here is 2^115.9.

Just asking since i thought this should be a lower bound.